OSPSuite.Presentation.Importer/Core/IDataSourceFileParser.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OSPSuite.Infrastructure.Import.Services;
+using OSPSuite.Presentation.Importer.Core.DataSourceFileReaders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OSPSuite.Presentation.Importer.Core
+{
+   public interface IDataSourceFileParser
+   {
+      IDataSourceFile For(string path);
+   }
+
+   public class DataSourceFileParser : IDataSourceFileParser
+   {
+      private readonly ICsvDataSourceFile csvDataSourceFile;
```

Comment:
This does not follow our coding conventions. private members are _xxx

Reply:
Done